modules/network-observability-multitenancy.adoc

```diff
@@ -3,22 +3,43 @@
 // network_observability/installing-operators.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="network-observability-multi-tenancy{context}"]
+[id="network-observability-multi-tenancy_{context}"]
 = Enabling multi-tenancy in Network Observability
-Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project admins. Project admins who have limited access to some namespaces can access flows for only those namespaces.
+Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project administrators. Project administrators who have limited access to some namespaces can access flows for only those namespaces. 
```

Comment:
Just want to put this here because its going to look weird, but according to IBM style guide, I have to use "and or or" and not "and/or" per but I will take this suggestion, as long as the project admin stuff isn't important anymore. I agree I should say Prometheus here too. 
> Do not use the construction and/or, which can be ambiguous and cause problems for translation. Instead, consider whether both conjunctions are needed; if not, use and or or. If both conjunctions are needed, use the construction option A, option B, or both.